pkg/action/failover.go

```diff
@@ -400,3 +410,89 @@ func (ac *ActionController) createSummary(action *storkv1.Action, ns string, sta
 	}
 	return failoverSummary, failbackSummary
 }
+
+func (ac *ActionController) performLastMileMigrationDuringFailover(action *storkv1.Action) {
+	if action.Status.Status == storkv1.ActionStatusSuccessful {
+		action.Status.Stage = storkv1.ActionStageScaleUpDestination
+		action.Status.Status = storkv1.ActionStatusInitial
+		action.Status.FinishTimestamp = metav1.Now()
+		ac.updateAction(action)
+		return
+	} else if action.Status.Status == storkv1.ActionStatusFailed {
+		// move to reactivate/rollback stage
+		action.Status.Stage = storkv1.ActionStageScaleUpSource
+		action.Status.Status = storkv1.ActionStatusInitial
+		action.Status.FinishTimestamp = metav1.Now()
+		ac.updateAction(action)
+		return
+	}
+
+	migrationScheduleReference := action.Spec.ActionParameter.FailoverParameter.MigrationScheduleReference
+
+	migrationSchedule, err := storkops.Instance().GetMigrationSchedule(migrationScheduleReference, action.Namespace)
```

Comment:
Are we not doing the static copy check or because it has crossed the validation check we are ignoring it?

Reply:
Yes, that has been already covered as part of the validations in InitialStage. So, I didn't feel the need to repeat that check